src/test/java/com/github/movimentocodar/maquinadecafe/maquinadecafe/creditos/GerenciadorCreditoTest.java

```diff
@@ -0,0 +1,89 @@
+package com.github.movimentocodar.maquinadecafe.maquinadecafe.creditos;
+
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GerenciadorCreditoTest {
+    GerenciadorCredito gerenciadorCredito = new GerenciadorCredito();
+
+    @Test
+    void testarAdicionarCreditoCedulaValido() {
+        List<BigDecimal> notasValidas;
+        notasValidas = Arrays.asList(
```

Comment:
crie e atribua na mesma linha

Reply:
vc poderia adicionar crédito em cada linha tbm ao invés de usar um for. Nesse caso o código ficou mais complexo do que precisava